shared/django_apps/bundle_analysis_app/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+# Added to avoid 'doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS' error
+# Needs to be called the same as the API app
```

Comment:
Don't think we need this comment, makes it sound like we had a BA app in API before but this only applies to django apps we had in API that then moved to Shared